lib/winapi/winioctl.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __WINIOCTL_H__
+#define __WINIOCTL_H__
+
+#include <xboxkrnl/xboxdef.h>
+
+typedef struct _FILE_OBJECTID_BUFFER {
+  BYTE  ObjectId[16];
+  union {
+    struct {
+      BYTE BirthVolumeId[16];
+      BYTE BirthObjectId[16];
+      BYTE DomainId[16];
+    } DUMMYSTRUCTNAME;
+    BYTE ExtendedInfo[48];
+  } DUMMYUNIONNAME;
```

Comment:
Nit: Consider using a "real" name.

Reply:
I used the names from the (public domain) header in mingw32 and am not sure that I have an idea for more descriptive names. Happy to change if you have a suggestion though.

For reference: https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/host/x86_64-w64-mingw32-4.8/+/refs/heads/studio-1.0-release/x86_64-w64-mingw32/include/winioctl.h#1641

Reply:
Fair enough, I didn't realize you had a reason for the dummy names. :)